pom.xml

```
@@ -8,8 +8,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <!-- Project Coordinates -->
-    <groupId>org.apache.spark.scheduler.armada</groupId>
-    <artifactId>armada-cluster-manager_2.13</artifactId>
+    <groupId>io.armadaproject.armada</groupId>
+    <artifactId>armada-cluster-manager_2.12</artifactId>
```

We should default to 2.13, and this must conform to `scala.version` below, which is 2.13
```suggestion
    <artifactId>armada-cluster-manager_2.13</artifactId>
```